APIManager.js

```diff
@@ -0,0 +1,80 @@
+
+class APIManager {
+    constructor() {
+        this.data = {}
+    }
+
+    getUserAndFriends(){
+        $.ajax({
+            method: 'GET',
+            url: `https://randomuser.me/api/?results=7`,
+            success: (response) => {
+                const users = response.results.map(p => { return {firstName: p.name.first, lastName: p.name.last, city: p.location.city, state: p.location.state, photo: p.picture.medium}})
+                this.data.mainUser = users[0]
+                this.data.friends = users.splice(1, 6)
+            },
+            error: function (xhr, text, error) {
+                console.log('error');
+            }
+        });     
+    }
+
+    getFavoriteQuote(){
+        $.ajax({
+            method: 'GET',
+            url: `https://api.kanye.rest`,
+            success: (response) => {
+                this.data.quote = response.quote
+            },
+            error: function (xhr, text, error) {
+                alert('error');
+            }
+        })
+    }
+
+    getRandomId = function() {
+        const randomPosition = Math.floor(Math.random() * 80); 
```

Comment:
you could just do one line in this function 
`return Math.floor(Math.random() * 80)`

you did not need to assign it a variable